test/test_list.sh

```diff
@@ -13,24 +13,22 @@ echo "### List local versions"
 cleanup || error_and_die "Cleanup failed?!"
 
 versions=(
-  0.38.12
-  0.37.4
-  0.36.11
-  0.33.0
-  0.29.7
+  0.54.11
+  0.54.10
+  0.54.9
+  0.54.8
+  0.49.1
 )
 for v in "${versions[@]}"; do
   tgenv install ${v} || error_and_proceed "Install of version ${v} failed"
 done
-tgenv use 0.38.12
+tgenv use 0.54.11
 
 result="$(tgenv list)";
 expected="$(cat << EOS
-* 0.38.12 (set by $(tgenv version-file))
-  0.37.4
-  0.36.11
-  0.33.0
-  0.29.7
+* 0.54.11 (set by $(tgenv version-file))
+  0.50.17
+  0.49.1
```

Comment:
These need to match lines 16-20.

The versions we install in versions=() are he versions that will be listed here.